statuspage/js/s3.js

```diff
@@ -70,7 +70,7 @@ checkup.storage = (function() {
 				doneCallback(checksLoaded);
 			} else {
 				for (var i = 0; i < list.length; i++) {
-					let url;
+					var url;
```

Comment:
It was breaking Safari, I don’t know if OS X has screwy js evaluation, please consider it just a suggestion.  I apologize for attaching it to the pull for the documentation update.


> On Aug 20, 2016, at 12:10 PM, Matt Holt <notifications@github.com> wrote:
> 
> In statuspage/js/s3.js <https://github.com/sourcegraph/checkup/pull/17#discussion_r75581874>:
> 
> > @@ -70,7 +70,7 @@ checkup.storage = (function() {
> >  				doneCallback(checksLoaded);
> >  			} else {
> >  				for (var i = 0; i < list.length; i++) {
> > -					let url;
> > +					var url;
> Why should url be scoped to the function and not just the for loop?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/sourcegraph/checkup/pull/17/files/3fc10ef9f1def0720bca74ed10642f3e74daf2c4#r75581874>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AT4YpcM9nOTQgmE95x0bEFJsGH_2h1_jks5qhzUSgaJpZM4Jo00l>.
>